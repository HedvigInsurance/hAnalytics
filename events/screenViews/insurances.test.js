const validateEvent = require("../../commons/validateEvent.js")

test('sets has_accident_insurance correctly for SE_ACCIDENT', () => {
    const event = validateEvent(__filename, {}, {
        contracts: [
            {
                typeOfContract: "SE_ACCIDENT"
            }
        ]
    })

    expect(event).toMatchSnapshot()
});

test('sets has_accident_insurance correctly for DK_ACCIDENT', () => {
    const event = validateEvent(__filename, {}, {
        contracts: [
            {
                typeOfContract: "DK_ACCIDENT"
            }
        ]
    })

    expect(event).toMatchSnapshot()
});

test('sets has_home_insurance correctly for SE_APARTMENT', () => {
    const event = validateEvent(__filename, {}, {
        contracts: [
            {
                typeOfContract: "SE_APARTMENT"
            }
        ]
    })

    expect(event).toMatchSnapshot()
});

test('sets has_home_insurance correctly for SE_HOUSE', () => {
    const event = validateEvent(__filename, {}, {
        contracts: [
            {
                typeOfContract: "SE_HOUSE"
            }
        ]
    })

    expect(event).toMatchSnapshot()
});

test('sets has_home_insurance correctly for NO_HOME_CONTENT_OWN', () => {
    const event = validateEvent(__filename, {}, {
        contracts: [
            {
                typeOfContract: "NO_HOME_CONTENT_OWN"
            }
        ]
    })

    expect(event).toMatchSnapshot()
});