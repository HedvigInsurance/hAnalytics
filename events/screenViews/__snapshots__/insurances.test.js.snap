// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sets has_accident_insurance correctly for SE_ACCIDENT 1`] = `
Object {
  "name": "screen_view_insurances",
  "variables": Object {
    "has_accident_insurance": true,
    "has_home_insurance": false,
  },
}
`;

exports[`sets has_home_insurance correctly for NO_HOME_CONTENT_OWN 1`] = `
Object {
  "name": "screen_view_insurances",
  "variables": Object {
    "has_accident_insurance": false,
    "has_home_insurance": true,
  },
}
`;

exports[`sets has_home_insurance correctly for SE_APARTMENT 1`] = `
Object {
  "name": "screen_view_insurances",
  "variables": Object {
    "has_accident_insurance": false,
    "has_home_insurance": true,
  },
}
`;

exports[`sets has_home_insurance correctly for SE_HOUSE 1`] = `
Object {
  "name": "screen_view_insurances",
  "variables": Object {
    "has_accident_insurance": false,
    "has_home_insurance": true,
  },
}
`;
